srcmedia/ts/members-map.ts

```diff
@@ -12,8 +12,12 @@ type Address = {
     start_date?: string,
     end_date?: string,
 }
-declare const addressData: Array<Address>
-declare const libraryAddress: Address|null
+
+const addressDataElement = document.getElementById('address-data') as HTMLElement
+const addressData = JSON.parse(addressDataElement.textContent || '') as Array<Address>
+
+const libraryAddressDataElement = document.getElementById('library-address') as HTMLElement
+const libraryAddress = JSON.parse(libraryAddressDataElement.textContent || '') as Address|null
```

Comment:
I think a more complete eventual solution for these is to have a shared error service that logs to console (and maybe also updates the search page UI with an error message), and then put them in `try()` blocks or conditionals so we know when they fail. But these casts are fine for now and we use them in plenty of other places

Reply:
I got used to using [`Option<T>` and `Result<T, E>`](https://learning-rust.github.io/docs/e3.option_and_result.html) in Rust and now kinda want them everywhere, so I might try implementing that in typescript as well

Reply:
When I implemented it the third time, it struck me as something that we might want a reusable solution for at some point.